js/app.js

```diff
@@ -0,0 +1,83 @@
+
+//question 1
+
+let obj ={
+	insideObj: {
+		key1: 'yo';
+		key2: 12
+	}
+
+}
+
+console.log(insideObj.key1())
```

Comment:
this is trying to call a method called key1 in an obj called insideObj--that method doesn't exist and this is not what the question is asking for..